docs/content/latest/yugabyte-cloud/develop/hasuracloud.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Hasura Cloud
+headerTitle: Connect Hasura Cloud to Yugabyte Cloud
+linkTitle: Hasura Cloud
+description: Connect Hasura Cloud to Yugabyte Cloud.
+menu:
+  latest:
+    identifier: hasura-cloud
+    parent: yugabyte-cloud
+    weight: 700
+isTocNested: true
+showAsideToc: true
+---
+
+Use the [Hasura GraphQL Engine](https://hasura.io) with Yugabyte Cloud to power your GraphQL applications with a distributed SQL database.
+
+The following example demonstrates how to deploy an application on both Hasura Cloud and Yugabyte Cloud using the Hasura sample Realtime Poll application. This application was built using React and is powered by Hasura GraphQL Engine backed by a Yugabyte Cloud YugabyteDB cluster. It has an interface for users to cast a vote on a poll and the results are updated in the on-screen bar chart in real time. 
+
+## Prerequisites
+
+The example has the following prerequisites.
+
+### Yugabyte Cloud account and Free Tier cluster
+
+Sign up for Yugabyte Cloud on the [Sign Up page](https://cloud.yugabyte.com/register).
+
+Once registered, create a Free Tier cluster by following the steps in [Create clusters](../../create-clusters/).
+
+The database cluster has a default database called `yugabyte` and administrator account called `admin` that we will use in our connection with our Hasura project. You will need the following cluster details:
+
+* password
+* IP address
+* port number
+
+To get these details, in the Yugabyte Cloud Console:
+
+1. Click **Go to cluster** to display the cluster console.
+
+1. Click **Connect**.
+
+1. **Copy** and save the details. These are provided in the form `PGPASSWORD=password ./bin/ysqlsh -h ip_address -p port -U admin -d yugabyte`, where password, ip_address, and port correspond to the values.
+
+### Hasura Cloud account
+
+To create a Hasura Cloud account, sign up at <https://cloud.hasura.io>.
+
+For details on using Hasura Cloud, see the [Hasura Cloud documentation](https://hasura.io/docs/latest/graphql/cloud/index.html).
+
+### Hasura CLI
+
+We will apply database migrations to our Hasura project using Hasura CLI v.2.0 Beta 2. To install Hasura CLI, refer to [Installing the Hasura CLI](https://hasura.io/docs/latest/graphql/core/hasura-cli/install-hasura-cli.html#install-hasura-cli).
+
+Once installed, update to the v.2 Beta.
+
+```sh
+$ hasura update-cli --version v2.0.0-beta.2
+```
+
+### Realtime Poll application
+
+The Realtime Poll application is available from the Yugabyte GraphQL Apps repo.
+
+```sh
+$ git clone https://github.com/yugabyte/yugabyte-graphql-apps
+cd yugabyte-graphql-apps/realtime-poll
+```
+
+## Create a Hasura project and connect to YugabyteDB
+
+To create a project in Hasura Cloud:
+
+1. From the Hasura Cloud dashboard, under **Projects**, click **New Project**. 
+
+1. Select **Free Tier**, leave the default region, and enter a project name of “yb-realtime-poll”.<br><br>
```

Comment:
Putting the breaks on their own line just makes them easier to notice. And self-closing is good form :)
```suggestion
1. Select **Free Tier**, leave the default region, and enter a project name of “yb-realtime-poll”.

    <br/><br/>
```